### DISCUSSION TITLE
What should the development containers provide

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#3708] [General]
The database, and the filesystem artifacts, are large.

The development containers currently provide a full (though scrubbed) database dump - there may be tables that we can populate sparsely by default and provide a way to pull full data as needed.

The containers provide very little of the filesystem artifacts by default - there's a command that would let you sync, for instance, the id-archive, which at the moment is 12GB. 

We should discuss what filesystem artifacts should be provided by default, and what should be added to the container to make fetching others as they are needed simpler.

**Comment by larseggert** [comment]
It's 2022. 12GB is not a problem.

**Reply by rjsparks** [reply]
Eh - not sure I can agree - time to move things around becomes an impediment, and we have an existence proof from the codesprint that some people don't have that many resources at their disposal. And we should keep an eye towards not making our contribution to global digital waste worse.
